h2o-hadoop/scripts/Jenkins/Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/groovy
+
+def AGENT_LABEL = 'mr-0xc10'
```

Comment:
Any specific reason to fix agent?

Reply:
In theory, you can use label `docker`

Reply:
At the moment only because of testing reasons. Maybe it will be good to add another job which will build the docker image and the test job will only pull the image